dev-tools/packaging/packages.yml

```diff
@@ -16,6 +16,158 @@ shared:
     url: '{{.BeatURL}}'
     description: '{{.BeatDescription}}'
 
+  # agent specific
+  # Deb/RPM spec for community beats.
+  - &deb_rpm_agent_spec
+    <<: *common
+    post_install_script: '{{ elastic_beats_dir }}/dev-tools/packaging/files/linux/systemd-daemon-reload.sh'
+    files:
+      /usr/share/{{.BeatName}}/bin/{{.BeatName}}{{.BinaryExt}}:
+        source: build/golang-crossbuild/{{.BeatName}}-{{.GOOS}}-{{.Platform.Arch}}{{.BinaryExt}}
+        mode: 0755
+      /usr/share/{{.BeatName}}/LICENSE.txt:
+        source: '{{ repo.RootDir }}/LICENSE.txt'
+        mode: 0644
+      /usr/share/{{.BeatName}}/README.md:
+        template: '{{ elastic_beats_dir }}/dev-tools/packaging/templates/common/README.md.tmpl'
+        mode: 0644
+      /usr/share/{{.BeatName}}/.build_hash.txt:
+        content: >
+          {{ commit }}
+        mode: 0644
+      /etc/{{.BeatName}}/agent.reference.yml:
+        source: 'agent.reference.yml'
+        mode: 0644
+      /etc/{{.BeatName}}/agent.yml:
+        source: 'agent.yml'
+        mode: 0600
+        config: true
+      /usr/share/{{.BeatName}}/bin/{{.BeatName}}-god:
+        source: build/golang-crossbuild/god-{{.GOOS}}-{{.Platform.Arch}}
+        mode: 0755
+      /usr/bin/{{.BeatName}}:
+        template: '{{ elastic_beats_dir }}/dev-tools/packaging/templates/linux/beatname.sh.tmpl'
+        mode: 0755
+      /lib/systemd/system/{{.BeatServiceName}}.service:
+        template: '{{ elastic_beats_dir }}/dev-tools/packaging/templates/linux/systemd.unit.tmpl'
+        mode: 0644
+      /etc/init.d/{{.BeatServiceName}}:
+        template: '{{ elastic_beats_dir }}/dev-tools/packaging/templates/{{.PackageType}}/init.sh.tmpl'
+        mode: 0755
+      /etc/{{.BeatName}}/beats/filebeat-{{ beat_version }}-{{.GOOS}}-{{.AgentArchName}}.tar.gz:
+        source: '{{ elastic_beats_dir }}/x-pack/filebeat/build/distributions/filebeat-{{ beat_version }}-{{.GOOS}}-{{.AgentArchName}}.tar.gz'
+        mode: 0644
+      /etc/{{.BeatName}}/beats/metricbeat-{{ beat_version }}-{{.GOOS}}-{{.AgentArchName}}.tar.gz:
+        source: '{{ elastic_beats_dir }}/x-pack/metricbeat/build/distributions/metricbeat-{{ beat_version }}-{{.GOOS}}-{{.AgentArchName}}.tar.gz'
+        mode: 0644
+
+
+  # MacOS pkg spec for community beats.
+  - &macos_agent_pkg_spec
+    <<: *common
+    extra_vars:
+      # OS X 10.8 Mountain Lion is the oldest supported by Go 1.10.
+      # https://golang.org/doc/go1.10#ports
+      min_supported_osx_version: 10.8
```

Comment:
our current build system is based on go1.13. Min macOS version is 1.11

Reply:
@urso its a copy-paste of the existing packages task defined in https://github.com/elastic/beats/blob/6034c9ab63c41848a36a528a7d27165c1419ea34/dev-tools/packaging/packages.yml#L222-L225

I will update mine to 1.11 and make a PR for the other part in master.